src/runtime/virtcontainers/clh.go

```diff
@@ -947,12 +947,6 @@ func (clh *cloudHypervisor) launchClh() (int, error) {
 		args = append(args, "-v")
 	}
 
-	// Disable the 'seccomp' option in clh for now.
-	// In this way, we can separate the periodic failures caused
-	// by incomplete `seccomp` filters from other failures.
-	// We will bring it back after completing the `seccomp` filter.
-	args = append(args, "--seccomp", "false")
```

Comment:
I am in very much in favor of enabling seccomp for cloud-hypervisor. However we did run into issues last time this feature was enabled for certain workloads to an overly restrictive list of system calls being restricted. 
In favor of adopting this feature for workloads, I think it may be worthwhile to have this as a config option initially set to true by default to easily check for any issues. Eventually, we can remove the config option. 
I'd like to see what others think as well.

Reply:
I like this idea and will update the PR in few days unless there are objections from the community.